.goreleaser.yml

```diff
@@ -0,0 +1,43 @@
+before:
+  hooks:
+    - go mod download
+builds:
+  - env:
+      - CGO_ENABLED=0
+    id: k8gb 
+    main: ./main.go
+    binary: bin/manager
+    goos:
+      - linux
+    goarch:
+      - amd64
+      - arm64
+dockers:
+- image_templates:
+  - "absaoss/k8gb:{{ .Tag }}-amd64"
+  use_buildx: false
+  dockerfile: Dockerfile-gr-amd64
+  build_flag_templates:
+  - "--platform=linux/amd64"
+- image_templates:
+  - "absaoss/k8gb:{{ .Tag }}-arm64"
+  use_buildx: false
+  goarch: arm64
+  dockerfile: Dockerfile-gr-arm64
+  build_flag_templates:
+  - "--platform=linux/arm64"
+docker_manifests:
+- name_template: absaoss/k8gb:{{ .Tag }}
+  image_templates:
+  - absaoss/k8gb:{{ .Tag }}-amd64
+  - absaoss/k8gb:{{ .Tag }}-arm64
+checksum:
+  name_template: 'checksums.txt'
+snapshot:
+  name_template: "{{ .Tag }}-next"
+changelog:
+  sort: asc
+  filters:
+    exclude:
+      - '^docs:'
+      - '^test:'
```

Comment:
Why exclude them?

Reply:
This is upstream behaviour and ignored by providing --release-notes flag, https://goreleaser.com/customization/release/#custom-release-notes

Reply:
I get it, just don't think we should exclude docs or tests from the changelog :)